### ISSUE TITLE
Small docs error

### ISSUE THREAD

**Issue by dirkdevriendt** [homeroom-live/graphql-middleware-apollo-upload-server#32]
Currently the docs suggest the following S3 init:
```js
const client = new S3({
  accessKeyId: __S3_KEY__,
  secretAccessKey: __S3_SECRET__,
  Bucket: __S3_BUCKET__,
})
```
This should, I think, have the Bucket in a params object:
```js
const client = new S3({
  accessKeyId: __S3_KEY__,
  secretAccessKey: __S3_SECRET__,
  params: { Bucket: __S3_BUCKET__ },
})
```
Love your work on the GraphQL middelwares, by the way, so big thanks for the effort!

**Comment by maticzav** [comment]
Hey @dirkdevriendt 👋,

Thank you for such great feedback 🎉. I will take your word for it and update the docs, thank you for noticing it. Would you mind creating a PR with the fix? 🙂

**Comment by dirkdevriendt** [comment]
ok, done!

**Comment by maticzav** [comment]
This has been resolved with #34
